### PR TITLE
new FAQ entry DE and EN: International phone numbers

### PR DESCRIPTION
**Pull request by dsarkar**
Closes https://github.com/corona-warn-app/cwa-website/issues/994

### CODE REVIEW

**Comment by dsarkar** [comment]
EN 
image

DE
image

File: src/data/faq.json [hunk 1]
```diff
@@ -28,7 +28,19 @@
...
                             "The various feedback channels of the <a href='https://www.coronawarn.app/de/community/'>Corona-Warn-App-Community<a/> will help you with your questions about the open source project of the Corona-Warn-App."
                         ]
+                    },
+                    {
+                        "title": "How to reach us from abroad - International phone numbers",
+                        "anchor": "international_phone_numbers",
+                        "active": true,
+                        "textblock": [
+                            "From abroad you can reach us at the following international phone numbers:",
+                            "Technical hotline: +49 30 498 75401 available Monday to Saturday from 7 a.m. to 10 p.m. (except on national holidays)",
```
Comment (heinezen) [inline src/data/faq.json:39]: > 7 a.m. to 10 p.m.

Should mention that this is local time in the GMT+1 timezone?

File: src/data/faq_de.json [hunk 2]
```diff
@@ -28,6 +28,17 @@
                             "Als Medienvertreter richten Sie bitte Ihre Anfragen an <a href='mailto:press@sap.com'>press@sap.com</a>.",
                             "Die verschiedenen Feedback-Kanäle der <a href='https://www.coronawarn.app/de/community/'>Corona-Warn-App-Community<a/> helfen Ihnen bei Fragen zum Open Source Projekt der Corona-Warn-App."
                         ]
+                    },
+                    {
+                        "title": "So erreichen Sie uns aus dem Ausland - Internationale Telefonnummern",
+                        "anchor": "international_phone_numbers",
+                        "active": true,
+                        "textblock": [
+                            "Vom Ausland aus erreichen Sie uns unter folgenden internationalen Telefonnummern:",
```
Comment (heinezen) [inline src/data/faq_de.json:38]: "Vom Ausland" -> "Außerhalb Deutschlands" oder "Außerhalb des deutschen Telefonnetzes"

File: src/data/faq_de.json [hunk 3]
```diff
@@ -28,6 +28,17 @@
...
                             "Die verschiedenen Feedback-Kanäle der <a href='https://www.coronawarn.app/de/community/'>Corona-Warn-App-Community<a/> helfen Ihnen bei Fragen zum Open Source Projekt der Corona-Warn-App."
                         ]
+                    },
+                    {
+                        "title": "So erreichen Sie uns aus dem Ausland - Internationale Telefonnummern",
+                        "anchor": "international_phone_numbers",
+                        "active": true,
+                        "textblock": [
+                            "Vom Ausland aus erreichen Sie uns unter folgenden internationalen Telefonnummern:",
+                            "Technische Hotline: +49 30 498 75401 erreichbar Montag bis Samstag von 7 bis 22 Uhr (außer an bundesweiten Feiertagen)",
```
Comment (heinezen) [inline src/data/faq_de.json:39]: local time GMT+1? (see other comment)

File: src/data/faq.json [hunk 4]
```diff
@@ -28,7 +28,19 @@
...
+                            "TAN hotline: +49 30 498 75402 available 24 hours a day, seven days a week",
+                            "Note: These numbers are <b>not free of charge</b> - the cost of the international call is based on the local contract with the user's provider."
```
Comment (heinezen) [inline src/data/faq.json:41]: "the user's provider" -> "your provider"

File: src/data/faq_de.json [hunk 5]
```diff
@@ -28,6 +28,17 @@
...
+                            "TAN-Hotline: +49 30 498 75402 erreichbar 24 Stunden und sieben Tage die Woche",
+                            "Hinweis: Diese Rufnummern sind <b>nicht kostenfrei</b> - die Kosten für den internationalen Anruf richten sich nach dem lokalen Vertrag mit dem Provider des Anwenders."
```
Comment (heinezen) [inline src/data/faq_de.json:41]: "mit dem Provider des Anwenders" -> "Ihrem Provider"

**Review by heinezen — CHANGES_REQUESTED** [review]
Getting there

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json:39]: ok, I guess most user will not understand GMT+1 timezone.

Maybe:
German local time?

File: src/data/faq_de.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq_de.json:38]: Außerhalb Deutschlands

ok !

File: src/data/faq.json (hunk 4 above)
Comment (dsarkar) [reply src/data/faq.json:41]: ok

Comment (dsarkar) [reply src/data/faq.json:41]: actually, remove also the word **local** ? ... based on the contract ...

File: src/data/faq_de.json (hunk 5 above)
Comment (dsarkar) [reply src/data/faq_de.json:41]: auch hier - remove **lokal** ??

File: src/data/faq.json (hunk 1 above)
Comment (heinezen) [reply src/data/faq.json:39]: yeah that should be okay

File: src/data/faq_de.json (hunk 5 above)
Comment (heinezen) [reply src/data/faq_de.json:41]: yes "lokal" is unnecessary

Comment (dsarkar) [reply src/data/faq_de.json:41]: ok

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json:39]: ok

File: src/data/faq.json [hunk 6]
```diff
@@ -22,13 +22,26 @@
                         "textblock": [
                             "The technical hotline (telephone 0800 7540001) will help you with technical questions about the Corona-Warn-App, for instance if you have problems with risk identification. You can reach the technical hotline from Monday to Saturday from 7 a.m. to 10 p.m. (except on German public holidays), and it is free of charge for you within Germany.",
                             "The TAN hotline (telephone 0800 7540002) will help you to enter the TAN if you have a positive test result and if you have not received a TAN or document with a QR code. The TAN hotline is available 24 hours a day, seven days a week. The call is free of charge.",
+                            "See also <a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>How to reach us from abroad </a>.",
```
Comment (heinezen) [inline src/data/faq.json:25]: Whitespace before `</a>` needs to be removed

File: src/data/faq_de.json [hunk 7]
```diff
@@ -22,12 +22,24 @@
                         "textblock": [
                             "Die technische Hotline (Telefon 0800 7540001) hilft Ihnen bei technischen Fragen rund um die Corona-Warn-App, zum Beispiel bei Problemen mit der Risiko-Ermittlung. Sie erreichen sie von Montag bis Samstag von 7 bis 22 Uhr (außer an bundesweiten Feiertagen), und sie ist für Sie innerhalb Deutschlands kostenfrei.",
                             "Die TAN-Hotline (Telefon 0800 7540002) hilft Ihnen bei der Eingabe der TAN, wenn Sie ein positives Testergebnis haben, und wenn Sie keine TAN oder kein Dokument mit QR-Code erhalten haben. Die TAN-Hotline ist 24 Stunden täglich an sieben Tagen in der Woche erreichbar. Der Anruf ist kostenfrei.",
+                            "Siehe auch <a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>So erreichen Sie uns aus dem Ausland</a>.",
```
Comment (heinezen) [inline src/data/faq_de.json:25]: "aus dem Ausland" -> "außerhalb Detschlands"

**Review by heinezen — CHANGES_REQUESTED** [review]

File: src/data/faq.json (hunk 6 above)
Comment (dsarkar) [reply src/data/faq.json:25]: ok

File: src/data/faq_de.json (hunk 7 above)
Comment (dsarkar) [reply src/data/faq_de.json]: ok

**Pull request merged**